### PR TITLE
docs: fix spelling mistakes

### PR DESCRIPTION
**Pull request by spnraju**
## Description

- This pull request addresses the issue described here: #<?>

In this pull request, I have fixed the wrongly spelt words in the documentation.

> _For features/enhancements:_
 - [x] I have added/updated the relevant references in the [documentation](https://github.com/wix/Detox/tree/master/docs) files.

> _For API changes:_
 - [ ] I have made the necessary changes in the [types index](https://github.com/wix/Detox/blob/master/detox/index.d.ts) file.

### CODE REVIEW

**Comment by noomorph** [comment]
Wow, that's been quite a good amount of proofreading. Thanks!!

image

**Comment by spnraju** [comment]
@noomorph Thank you for for correcting some of the mistakes! 😃

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The most difficult part of automated testing on mobile is the tip of the testing
 * **Runs on Devices** (not yet supported on iOS): Gain confidence to ship by testing your app on a device/simulator just like a real user.
 * **Automatically Synchronized:** Stops flakiness at the core by monitoring asynchronous operations in your app.
 * **Made For CI:** Execute your E2E tests on CI platforms like Travis without grief. 
-* **Test Runner Independent:** Use Jest, Mocha, AVA, or any other JavaScript test runner you like (spoiler: we have our favourite).
+* **Test Runner Independent:** Use Jest, Mocha, AVA, or any other JavaScript test runner you like (spoiler: we have our favorite).
```
Comment (d4vidi) [inline README.md:50]: https://www.grammarly.com/blog/favorite-favourite/

**Comment by d4vidi** [comment]
Lol thanks!

**Pull request merged**